<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLiveryFilterBar</class>
 <widget class="QFrame" name="CLiveryFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>44</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gl_CLiveryFilterBar">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="3">
    <widget class="QLabel" name="lbl_AirlineIcaoCode">
     <property name="text">
      <string>Airline:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_LiveryCode">
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lbl_Description">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="le_LiveryCode">
     <property name="placeholderText">
      <string>Livery code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="BlackGui::Filters::CFilterBarButtons" name="filter_Buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="le_Description"/>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="le_AirlineIcaoCode">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>ICAO</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="4">
    <widget class="QWidget" name="wi_Colors" native="true">
     <layout class="QHBoxLayout" name="hl_Colors">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Color">
        <property name="text">
         <string>Color (F/Tl):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BlackGui::CColorSelector" name="color_Fuselage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BlackGui::CColorSelector" name="color_Tail">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hs_ColorDistance">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Color distance</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QWidget" name="wi_Flags" native="true">
     <layout class="QHBoxLayout" name="hl_Flags">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_Colors">
        <property name="text">
         <string>Color liveries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Airlines">
        <property name="text">
         <string>Airline liveries</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Flags">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="7">
    <spacer name="hs_Row1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CColorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/colorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>blackgui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_LiveryCode</tabstop>
  <tabstop>le_AirlineIcaoCode</tabstop>
  <tabstop>le_Description</tabstop>
  <tabstop>cb_Colors</tabstop>
  <tabstop>cb_Airlines</tabstop>
  <tabstop>hs_ColorDistance</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
